### PR TITLE
Testing of the methods inside the enumerable module

### PR DESCRIPTION
**Pull request by muhenge**
## Tests done are
- Checking all methods without parameters,
- All methods with parameters,
- All the methods that have blocks and parameters at the same time  
- All methods without blocks and with blocks.

### CODE REVIEW

File: spec/enumerable_spec.rb [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    it 'all the values are equal than 1' do
+      expect([1, 1, 1, 1, 1].my_all?(1)).to eql(true)
+    end
+    it 'all the values are numbers' do
+      expect([8, 4, 3, 9, 5].my_all?(Numeric)).to eql(true)
+    end
+    it 'all the values are not Strings' do
+      expect([8, 4, 3, 9, 5].my_all?(String)).to eql(false)
+    end
+  end
```
Comment (kobiyoyo) [inline spec/enumerable_spec.rb]: - Lets write test for when reqex is given
- Also write for when no block or argument is given

File: spec/enumerable_spec.rb [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    it "there isn't any value greater or equal than 10" do
+      expect([8, 4, 3, 9, 5].my_any? { |x| x >= 10 }).to eql(false)
+    end
+    it 'at least one of the values is equal than 8' do
+      expect([8, 'd', 'e', 9, 5].my_any?(8)).to eql(true)
+    end
+    it "there isn't any value equal than 1" do
+      expect([3, 4, 5, 6, 7].my_any?(1)).to eql(false)
+    end
+  end
```
Comment (kobiyoyo) [inline spec/enumerable_spec.rb]: - Lets write test for when reqex is given
- Also write for when no block or argument is given
- Lastly lets write for when a class is passed as an argument

File: spec/enumerable_spec.rb [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    it 'the numbers are not positive' do
+      expect([20, 3, 6, 10].my_none? { |x| x > 0 }).to eql(false)
+    end
+    it 'there is not a number 7 inside the array' do
+      expect([20, 3, 6, 10].my_none?(7)).to eql(true)
+    end
+    it 'there is a number 6 inside the array' do
+      expect([20, 3, 6, 10].my_none?(6)).to eql(false)
+    end
+  end
```
Comment (kobiyoyo) [inline spec/enumerable_spec.rb]: - Lets write test for when reqex is given
- Also write for when no block or argument is given
- Lastly lets write for when a class is passed as an argument

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi  Team    
#### Great Job so far, Just few things to work on. 
### General
check the comments below
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi team:clap: Good job, but one kind of test recommended from the previous review is still not here**
- When a class is given as argument to **my_any?** and **my_none?**. You have a similar kind of test for my_all? You now have to add such test for my_any? and my_none?
Here's one from my_all?
```
      it 'all the values are not Strings' do
        expect([8, 4, 3, 9, 5].my_all?(String)).to eql(false)
      end
```
_**Kindly make this change and submit another code review request.**_

**Any questions? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### Status: Changes Requested.

**Review by Torres-ssf — APPROVED** [review]
### Project Approved :tada: :+1: 
Hi there team :hand: 
Congratulations on implementing testing for your enumarable methods. I can see that many tests were implemented and you also worked on the requested changes. :clap: 
This PR is approved. you can merge it now :smile: 

**TIP**
I notice the README file is missing one of the Authors

Happy coding :computer: 
**Sergio Torres**

**Pull request merged**